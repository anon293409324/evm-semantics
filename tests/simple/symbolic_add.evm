account:
-   id: 1
-   balance: 40
-   program: PUSH 10 ; PUSH #symbolicWord ; ADD
-   storage: 30 , 20 , 25

.Accounts


transaction:
-   to: 1
-   from: .AcctID
-   data:
-   value: 30
-   gasPrice: 1
-   gasLimit: 50

.Transactions
