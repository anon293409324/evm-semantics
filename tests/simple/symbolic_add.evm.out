<T> <k> . </k> <id> 1 </id> <pc> 3 </pc> <gas> 41 </gas> <wordStack> ( V0 +Int 10 ) : .WordStack </wordStack> <localMem> .Map </localMem> <gasPrice> 1 </gasPrice> <callStack> .CallStack </callStack> <accounts> <account> <acctID> 1 </acctID> <program> 0 |-> ( PUSH 10 ) 1 |-> ( PUSH V0 ) 2 |-> ADD </program> <storage> 0 |-> 30 1 |-> 20 2 |-> 25 </storage> <balance> 70 </balance> </account> </accounts> </T>
